src/main/java/SurrealChat.java

```diff
@@ -0,0 +1,139 @@
+     * @param inputString The entire command, inclusive of arguments if any.
+     * @return Output to be printed.
+     */
+    public String commandLogic(String inputString) {
+        String[] separatedWords = this.splitString(inputString);
+        String userCommand = separatedWords[0];
+        String restOfInput = "";
+
+        //Reorganise the remainder of input.
+        for (int i = 1; i < separatedWords.length; i++) {
+            restOfInput += separatedWords[i];
+        }
+        restOfInput = restOfInput.trim();
+
+        //Generate output
+        switch(userCommand) {
+        case "help":
+            return HelpMode.displayHelp(restOfInput);
+        case "list":
+            Command listCommand = new ListCommand();
+            String outputList = listCommand.execute(this.taskManagement);
+            return outputList;
+        case "todo":
+            Command addToDoCommand = new ToDoCommand(restOfInput);
+            String outputString = addToDoCommand.execute(this.taskManagement);
+            return outputString;
+        case "deadline":
+            Command addDeadlineCommand = new DeadlineCommand(restOfInput);
+            outputString = addDeadlineCommand.execute(this.taskManagement);
+            return outputString;
+        case "event":
+            Command addEventCommand = new EventCommand(restOfInput);
+            outputString = addEventCommand.execute(this.taskManagement);
+            return outputString;
+        case "edit":
+            Command editCommand = new EditCommand(restOfInput);
+            outputString = editCommand.execute(this.taskManagement);
+            return outputString;
+        case "done":
+            Command doneCommand = new DoneCommand(restOfInput);
+            outputString = doneCommand.execute(this.taskManagement);
+            return outputString;
+        case "delete":
+            Command deleteCommand = new DeleteCommand(restOfInput);
+            outputString = deleteCommand.execute(this.taskManagement);
+            return outputString;
+        case "find":
+            Command findCommand = new FindCommand(restOfInput);
+            outputString = findCommand.execute(this.taskManagement);
+            return outputString;
+        case "orang":
+            EasterEgg orangEasterEgg = new OrangEasterEgg();
+            outputString = orangEasterEgg.execute();
+            return outputString;
+        case "vegetal":
+            EasterEgg vegetalEasterEgg = new VegetalEasterEgg();
+            outputString = vegetalEasterEgg.execute();
+            return outputString;
+        default:
+            return "Command not recognised. Not stonks!\n";
+        }
+    }
+
+    /**
+     * Loads the tasks from file.
+     */
+    public void loadFile() {
+        try {
+            List<String> fileLines = this.fileManagement.loadTaskFile();
+            this.taskManagement.parseFileLines(fileLines);
+        } catch (IOException e) {
+            return;
```

Comment:
Perhaps you can write a comment to describe why the catch block returns nothing.